### DISCUSSION TITLE
Help with the settings for publishing

### DISCUSSION THREAD

**Discussion by skramovskiy** [airsdk/Adobe-Runtime-Support#1686] [General]
Good afternoon
I use Adobe Animate
after installing the AIR SDK 33.1.1.743 an error occurs when publishing the APK:

" Android Gradle-based package creation needs an Android SDK path to be provided [-platformsdk] "

how to fix it?
there are no such problems in version 33.1.1.259.

**Comment by mohammadfard1365** [comment]
add this tag in app-description.xml : 
 
`<BuildLegacyAPK>true</BuildLegacyAPK>
`

**Reply by skramovskiy** [reply]
unfortunately, it did not solve the problem.

**Reply by FliplineStudios** [reply]
If you only need an APK at the moment, you can add that BuildLegacyAPK tag to your app xml (and make sure it's within the `<android>` section, but outside of the manifestAdditions).

If you plan to submit a brand-new app to the Play Store though, you will need to use the aab format now instead of APK, and for that you will need to set up an Android SDK path like that error mentions.  There are some more details about that process here: https://airsdk.dev/docs/tutorials/platform/android/packaging-android-app-bundles

Since you're using Animate, I believe you would need to use the "adt.cfg" method mentioned in that tutorial to point to your Android SDK path, after you've downloaded and installed one separately.

**Comment by marchbold** [comment]
2 additional things need to be done now to get animate working:
- Install Android Studio (including the Android SDK); and
- set the path to the Android SDK in your adt.cfg file. 

Have a look at the install guide here for more information on installing Android Studio: https://airsdk.dev/docs/basics/install/macos#android-setup

Then to set the platform sdk with animate the easiest way is to edit the adt.cfg (`AIRSDK/lib/adt.cfg`) file and set the path to the android sdk, eg this is in my `adt.cfg`:

```
# For convenience, rather than using "-platformsdk" on the command line, these can be set up here:
AndroidPlatformSDK=/Users/marchbold/work/sdks/android/android-sdk-macosx
```
